ext/standard/uniqid.c

```diff
@@ -71,7 +72,11 @@ PHP_FUNCTION(uniqid)
 	 * digits for usecs.
 	 */
 	if (more_entropy) {
-		uniqid = strpprintf(0, "%s%08x%05x%.8F", prefix, sec, usec, php_combined_lcg() * 10);
+		double bytes;
+		if (php_random_bytes_silent(&bytes, sizeof(double)) == FAILURE) {
+			bytes = php_combined_lcg() * 10;
+		}
+		uniqid = strpprintf(0, "%s%08x%05x%.8F", prefix, sec, usec, bytes);
```

Comment:
This may likely generate strings in a different format if you're lucky and generate a NAN or so...

Reply:
Actually, no NAN needed, tests already show the problem:

> string(313) "5fdc535dc5838-31657434423802766756251491823837608490147380143897364946468120499868248688070997162348412820763960955954949255892557493142678733689837618721102246898520502380922020923630697671566381671530605217563575998370293070799806327032422345688791574258594719753295271835053247587386071232895948685312.00000000"

I guess php_combined_lcg() only returns a number between 0 and 1. We'd have to rescale random_bytes output for that.

Reply:
I made a silly mistake. double is not suitable for this use case. This has been fixed. Thanks!